windows/src/developer/TIKE/dialogs/UfrmVisualKeyboardImportKMX.pas

```diff
@@ -125,6 +127,12 @@ procedure TfrmVisualKeyboardImportKMX.FormShow(Sender: TObject);
  - Prepare for key processing                                                  -
  ------------------------------------------------------------------------------}
 
+procedure TfrmVisualKeyboardImportKMX.SetStatus(const msg: string);
+begin
+  lblStatus.Caption := msg;
```

Comment:
It's a proportional font, so it depends. We don't really care, because it's just for feedback as the process runs. Longer messages will be cropped. Could be helpful for diagnostics in case we get another issue like the one that was the trigger this fix.